src/libs/runtime-utils.js

```diff
@@ -233,14 +253,15 @@ export const getGalaxyComputeCost = app => {
  * - Disk cost is total for data (NFS) disk, metadata (postgres) disk, and boot disks (1 boot disk per nodepool)
  * - Size of a data disk is user-customizable. The other disks have fixed sizes.
  */
-export const getGalaxyDiskCost = dataDiskSize => {
+export const getGalaxyDiskCost = ({ size, diskType }) => {
   const metadataDiskSize = 10 // GB
   const defaultNodepoolBootDiskSize = 100 // GB
   const appNodepoolBootDiskSize = 100 // GB
 
   return getPersistentDiskCostHourly({
     status: 'Running',
-    size: dataDiskSize + metadataDiskSize + defaultNodepoolBootDiskSize + appNodepoolBootDiskSize
+    size: size + metadataDiskSize + defaultNodepoolBootDiskSize + appNodepoolBootDiskSize,
```

Comment:
```suggestion
    size: dataDiskSize + metadataDiskSize + defaultNodepoolBootDiskSize + appNodepoolBootDiskSize,
```

may be more consistent and clearer (in that `size` on the left and the right are different). I understand that it'll come at the cost of more verboseness at input deconstruction time; i.e.

```javascript
export const getGalaxyDiskCost = ({ size: dataDiskSize, diskType }) => {
```